src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporterHelperExtensions.cs

```diff
@@ -39,19 +41,61 @@ public static TracerProviderBuilder AddOtlpExporter(this TracerProviderBuilder b
             {
                 return deferredTracerProviderBuilder.Configure((sp, builder) =>
                 {
-                    AddOtlpExporter(builder, sp.GetOptions<OtlpExporterOptions>(), configure);
+                    AddOtlpExporter(builder, sp.GetOptions<OtlpExporterOptions>(), configure, sp);
                 });
             }
 
-            return AddOtlpExporter(builder, new OtlpExporterOptions(), configure);
+            return AddOtlpExporter(builder, new OtlpExporterOptions(), configure, serviceProvider: null);
         }
 
-        private static TracerProviderBuilder AddOtlpExporter(TracerProviderBuilder builder, OtlpExporterOptions exporterOptions, Action<OtlpExporterOptions> configure = null)
+        internal static void BuildHttpClientFactory(IServiceProvider serviceProvider, OtlpExporterOptions options, string httpClientName)
```

Comment:
Maybe move this method to `OtlpExporterOptionsExtensions.cs`?

Reply:
Updated